src/test/java/io/meeds/qa/ui/steps/definition/ManageSpaceStepDefinitions.java

```diff
@@ -379,4 +380,9 @@ public void uploadSpaceBanner(String fileName) {
     manageSpaceSteps.uploadSpaceBanner(fileName);
   }
 
-}
+  @And("^I remove application '(.*)' from the list application$")
```

Comment:
Same here, you can make the clicked button parameterized

```suggestion
  @And("^I click on '(.*)' option from application '(.*)' menu$")
```

Reply:
Done